src/inttest/resources/config/application.yml

```diff
@@ -43,6 +43,8 @@ faf-api:
     image-height: 20
   clan:
     website-url-format: "http://example.com/%s"
+  tutorial:
+    thumbnail-url-format: "http://test.com/faf/tutorials/%s/xyz"
```

Comment:
In `application.yml`: Nothing, because we don't want a default value
In `application-prod.yml`: `${TUTORIAL_THUMBNAIL_URL_FORMAT}` so we enforce the environment variable to be set
In `application-dev.yml`: `${TUTORIAL_THUMBNAIL_URL_FORMAT:content.test.faforever.com/faf/tutorials/thumbs/%s}` so we can set the environment variable, or else it uses a default value that's a best-guess and when the developer executes it locally

Reply:
Acctually it is correct in there as the application.yml is the one of the integration tests;)